components/BarcodeScanner.vue

```diff
@@ -242,27 +277,26 @@ const FORMAT_MAP = new Map([
   ['pdf417', Html5QrcodeSupportedFormats.PDF_417],
   ['qr_code', Html5QrcodeSupportedFormats.QR_CODE],
   ['upc_a', Html5QrcodeSupportedFormats.UPC_A],
-  ['upc_e', Html5QrcodeSupportedFormats.UPC_E]
+  ['upc_e', Html5QrcodeSupportedFormats.UPC_E],
 ]);
 
 // map from Web-native format to `Html5QrcodeSupportedFormats`
-function _mapFormats(formats) {
+export function _mapFormats(formats) {
   return formats.map(format => {
     const result = FORMAT_MAP.get(format);
     if(result === undefined) {
-      if(typeof result !== 'string' ||
-        !isNaN(Number.parseInt(result, 10))) {
+      if(typeof result !== 'string' || !isNaN(Number.parseInt(result, 10))) {
         throw new TypeError(
           `Unsupported format "${format}"; ` +
-          'a string supported by the "BarcodeFormat" enumeration ' +
-          'must be given, e.g., "qr_code", not a number.');
+            'a string supported by the "BarcodeFormat" enumeration ' +
+            'must be given, e.g., "qr_code", not a number.'
+        );
```

Comment:
Style nit... what was here before was correct, this change makes it incorrect style.

```suggestion
          'a string supported by the "BarcodeFormat" enumeration ' +
          'must be given, e.g., "qr_code", not a number.');
```